CMakeLists.txt

```diff
@@ -222,6 +223,10 @@ endif()
 if(ACTS_BUILD_EXAMPLES_GEANT4)
   find_package(Geant4 REQUIRED CONFIG COMPONENTS gdml)
 endif()
+if(ACTS_BUILD_EXAMPLES_GEOMODEL)
+  find_package( GeoModelCore PATHS ${GeoModel_INSTALL}  REQUIRED ${GeoModel_VERSION} )
+  find_package( GeoModelG4 PATHS ${GeoModel_INSTALL} REQUIRED ${GeoModel_VERSION} )
```

Comment:
The question is, do we bundle GeoModel? I think it's small enough that it could be viable, but not sure if it's necessary.